tests/components/ClaimStatus.test.tsx

```diff
@@ -18,39 +19,63 @@ function renderClaimStatusComponent(statusContent: ClaimStatusContent): string {
     .toJSON()
 }
 
-function testClaimStatus(scenarioType: ScenarioType, hasCertificationWeeksAvailable: boolean): string {
+function testClaimStatus(
+  scenarioType: ScenarioType,
+  hasCertificationWeeksAvailable: boolean,
+  userArrivedFromUioMobile: boolean,
+): string {
   const scenarioContent = getScenarioContent(apiGatewayStub(scenarioType, hasCertificationWeeksAvailable))
-  return renderClaimStatusComponent(scenarioContent.statusContent)
+  return renderClaimStatusComponent(scenarioContent.statusContent, userArrivedFromUioMobile)
```

Comment:
Note: I've added a lot of snapshots here, but I thought it didn't hurt to have extra/added coverage.